doc/README

```diff
@@ -6,19 +6,59 @@ Nov 25, 2011
 
 For Tuxmath-2.0.4
 
+Table of Contents
+------------
```

Comment:
Thanks, this is helpful. 

Tooling exists to generate tables of contents from the document's contents, so that it never gets out of sync. For instance, https://github.com/thlorenz/doctoc, but there are others. For your next PR, can I suggest using one of these? 😄